UnitsNet/UnitConverter.cs

```diff
@@ -321,24 +320,25 @@ public static bool TryConvert(double fromValue, Enum fromUnitValue, Enum toUnitV
         ///     c) To unit: Meter, Centimeter etc if Length is selected
         /// </summary>
         /// <param name="fromValue">
-        ///     Input value, which together with <paramref name="fromUnit" /> represents the quantity to
+        ///     Input value, which together with <paramref name="fromUnitName" /> represents the quantity to
         ///     convert from.
         /// </param>
         /// <param name="quantityName">The invariant quantity name, such as "Length". Does not support localization.</param>
-        /// <param name="fromUnit">The invariant unit enum name, such as "Meter". Does not support localization.</param>
-        /// <param name="toUnit">The invariant unit enum name, such as "Meter". Does not support localization.</param>
+        /// <param name="fromUnitName">The invariant unit enum name, such as "Meter". Does not support localization.</param>
+        /// <param name="toUnitName">The invariant unit enum name, such as "Meter". Does not support localization.</param>
         /// <example>double centimeters = ConvertByName(5, "Length", "Meter", "Centimeter"); // 500</example>
-        /// <returns>Output value as the result of converting to <paramref name="toUnit" />.</returns>
-        /// <exception cref="UnitNotFoundException">No units match the abbreviation.</exception>
+        /// <returns>Output value as the result of converting to <paramref name="toUnitName" />.</returns>
+        /// <exception cref="QuantityNotFoundException">
+        ///     Thrown when no quantity information is found for the specified quantity name.
+        /// </exception>
+        /// <exception cref="UnitNotFoundException">No units match the provided unit name.</exception>
         /// <exception cref="AmbiguousUnitParseException">More than one unit matches the abbreviation.</exception>
-        public static double ConvertByName(double fromValue, string quantityName, string fromUnit, string toUnit)
+        public static double ConvertByName(double fromValue, string quantityName, string fromUnitName, string toUnitName)
```

Comment:
Technically this is another breaking change..

Reply:
Only for reflection I believe.
Besides, we're targeting v6 so it's fine.